Toplean_Alexandru/week-02/Domain/Domain/PlaceOrderOp/PlaceOrderOp.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Infrastructure.Free;
+using LanguageExt;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using static Domain.Domain.PlaceOrderOp.PlaceOrderResult;
+
+namespace Domain.Domain.PlaceOrderOp
+{
+    internal class PlaceOrderOp : OpInterpreter<PlaceOrderCmd, IPlaceOrderResult, Unit>
+    {
+        public override Task<IPlaceOrderResult> Work(PlaceOrderCmd Op, Unit state)
+        {
+            (bool CommandIsValid, String Error) = Op.IsValid();
+            if (CommandIsValid)
+            {
+                AllHardcodedValues.HarcodedVals.Carts[Op.Client.SessionID].Status = Models.Cart.CartStatus.CartSubmitted;
```

Comment:
What happens if your persistence layer doesn't have that cart? 

https://twitter.com/billwagner/status/1251156770988232705